src/openrct2/windows/track_list.c

```diff
@@ -259,11 +265,7 @@ static void window_track_list_mouseup(rct_window *w, rct_widgetindex widgetIndex
 		break;
 	case WIDX_BACK:
 		window_close(w);
-		if (gScreenFlags & SCREEN_FLAGS_TRACK_MANAGER) {
-			window_close_by_number(WC_MANAGE_TRACK_DESIGN, w->number);
-			window_close_by_number(WC_TRACK_DELETE_PROMPT, w->number);
-			trackmanager_load();
-		} else {
+		if (gScreenFlags | SCREEN_FLAGS_TRACK_MANAGER) {
```

Comment:
Uhhh I'm not honestly sure what happened there - it wasn't what I was going for! Fixed in next commit.